PrivateClinicsWebNet.Application/Mapping/AppointmentCreationProfileMap.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AutoMapper;
+using PrivateClinicsWebNet.Application.DTOs.AppointmentsDTOs;
+using PrivateClinicsWebNet.BusinessLogic.Entities;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PrivateClinicsWebNet.Application.Mapping
+{
+    public class AppointmentCreationProfileMap : Profile
+    {
+        public AppointmentCreationProfileMap() 
+        {
+            CreateMap<CreateAppointmentDto, Appointment>()
+                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => Guid.NewGuid()))
+                .ForMember(dest => dest.ExternalId, opt => opt.MapFrom(src => src.ExternalId))
```

Comment:
You are allowed to not define mapping explicitly for fields which have the same name